stories/VoteParticipation.stories.tsx

```diff
@@ -8,7 +8,7 @@ export default {
 
 const Template: ComponentStory<typeof VoteParticipation> = () => (
   <div style={{ padding: "2rem", background: "var(--black)" }}>
-    <VoteParticipation />
+    <VoteParticipation apy="30.1" />
```

Comment:
you have a constant, can we use that? also how does this get overridden with the actual value